pkgs/games/gscrabble/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub
+, gtk3, gobjectIntrospection, wrapGAppsHook, gst_all_1
+, python3, gnome3, hicolor-icon-theme }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "gscrabble";
+  version = "2019-03-11";
+
+  src = fetchFromGitHub {
+    owner = "RaaH";
+    repo = "gscrabble";
+    rev = "4b6e4e151a4cd4a4f66a5be2c8616becac3f2a29";
+    sha256 = "0a89kqh04x52q7qyv1rfa7xif0pdw3zc0dw5a24msala919g90q2";
+  };
+
+  doCheck = false; # no tests available
+
+  buildInputs = with gst_all_1; [
+    gst-plugins-base gst-plugins-good gst-plugins-ugly gst-plugins-bad
+    hicolor-icon-theme gnome3.adwaita-icon-theme
+  ];
+
+  nativeBuildInputs = [ 
+    python3
+    gobjectIntrospection # populate GI_TYPELIB_PATH
+    wrapGAppsHook
+   ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    gtk3
```

Comment:
`gtk3` should not need to be propagated. It is just a regular library that will be picked up by `wrapGAppsHook`. `propagatedBuildInputs` are really only useful if you want something to propagate, or for Python libraries until `buildPythonPackages` gets a better dependency handling.